core/liquidity/v2/sla.go

```diff
@@ -149,7 +149,7 @@ func (e *Engine) doesLPMeetsCommitment(
 		price := o.Price.ToDecimal()
 		// this order is in range and does contribute to the volume on notional
 		if price.GreaterThanOrEqual(minPrice) && price.LessThanOrEqual(maxPrice) {
-			orderVolume := num.UintZero().Mul(markPrice, num.NewUint(o.TrueRemaining())).ToDecimal().Div(positionFactor)
+			orderVolume := num.UintZero().Mul(markPrice, num.NewUint(o.VisibleRemaining())).ToDecimal().Div(positionFactor)
```

Comment:
I have a feeling that "visible peak" in the AC just means `o.Remaining`? I also don't really know why only the visible peak counts, so I might ask protocol design.

Reply:
Spec is changed to mean all available volume, so reverting to the old code/